test/unit/sql/change-column.test.js

```diff
@@ -33,23 +33,24 @@ if (current.dialect.name !== 'sqlite') {
       after(function () {
         this.stub.restore();
       });
-
-      it('properly generate alter queries', () => {
-        return current.getQueryInterface().changeColumn(Model.getTableName(), 'level_id', {
-          type: DataTypes.FLOAT,
-          allowNull: false,
-        }).then(sql => {
-          expectsql(sql, {
-            ibmi: 'ALTER TABLE "users" ALTER COLUMN "level_id" SET DATA TYPE FLOAT NOT NULL',
-            mssql: 'ALTER TABLE [users] ALTER COLUMN [level_id] FLOAT NOT NULL;',
-            db2: 'ALTER TABLE "users" ALTER COLUMN "level_id" SET DATA TYPE FLOAT ALTER COLUMN "level_id" SET NOT NULL;',
-            mariadb: 'ALTER TABLE `users` CHANGE `level_id` `level_id` FLOAT NOT NULL;',
-            mysql: 'ALTER TABLE `users` CHANGE `level_id` `level_id` FLOAT NOT NULL;',
-            postgres: 'ALTER TABLE "users" ALTER COLUMN "level_id" SET NOT NULL;ALTER TABLE "users" ALTER COLUMN "level_id" DROP DEFAULT;ALTER TABLE "users" ALTER COLUMN "level_id" TYPE FLOAT;',
-            snowflake: 'ALTER TABLE "users" ALTER COLUMN "level_id" SET NOT NULL;ALTER TABLE "users" ALTER COLUMN "level_id" DROP DEFAULT;ALTER TABLE "users" ALTER COLUMN "level_id" TYPE FLOAT;',
+      if (current.dialect.name !== 'yugabytedb'){ // Alter Column type not supported yet in yugabytedb
```

Comment:
(code style) Prefer adding your comments before the if statement

```suggestion
      // Alter Column type not supported yet in yugabytedb
      if (current.dialect.name !== 'yugabytedb'){
```